docs/aro/ibm-cloud-paks-for-data/README.md

```diff
@@ -0,0 +1,278 @@
+# ARO IBM Cloud Paks 4 Data
+
+A Quickstart guide to deploying an Azure Red Hat OpenShift cluster with IBM Cloud Paks 4 Data.
+
+Author: [Kristopher White]
+
+## Video Walkthrough
+
+If you prefer a more visual medium, you can watch [Kristopher White] walk through this quickstart on [YouTube](https://youtu.be/iifsB-uuEFc).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VYfCltxoh40" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
this embed code is wrong as it links to Paul's video deploying Azure openshift, not your video,